rapids-cmake/cpm/versions.json

```diff
@@ -42,7 +42,7 @@
       "git_tag": "${version}"
     },
     "GTest": {
-      "version": "1.13.0",
+      "version": "1.15.2",
```

Comment:
Why not `1.16.0`?

Reply:
I'm happy to switch over to 1.16. I'll test that now.

Reply:
1.16 also worked fine for the build. I'll bump.